### PR TITLE
Package structure

### PR DESCRIPTION
**Pull request by mark-beiline**
* Updated `README.md`
* Added `package.json` file

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "com.disguise.renderstream",
+  "displayName": "Disguise RenderStream",
+  "version": "1.0.0",
+  "unity": "2023.2",
```
Comment (FrederickUnity) [inline package.json]: That's the minimal version?  Is 2023.2 released yet?

**Review by FrederickUnity — APPROVED** [review]
Since my knowledge of packages is 0, wait for Wen's feedback before commit :)

File: package.json (hunk 1 above)
Comment (mark-beiline) [reply package.json]: Changed to 2022.2.6f1 which I've used a lot during dev with no issues. If we go too far back (ex 2022.2.0b16) some of the API is missing so I'll keep it at 6f1 for now.

**Review by wenzhang-unity — APPROVED** [review]
Looks good to me!

**Pull request merged**